store/sqlstore/plugin_store.go

```diff
@@ -53,7 +53,7 @@ func (ps SqlPluginStore) SaveOrUpdate(kv *model.PluginKeyValue) (*model.PluginKe
 				// If the error is from unique constraints violation, it's the result of a
 				// valid race and we can report success. Otherwise we have a real error and
 				// need to return it
-				if !IsUniqueConstraintError(err, []string{"PRIMARY", "PluginId", "Key", "PKey"}) {
+				if !IsUniqueConstraintError(err, []string{"PRIMARY", "PluginId", "Key", "PKey", "pkey"}) {
```

Comment:
Yikes, this is a concerning issue if Postgres wasn't reliably distinguishing this case from an error. Can you elaborate why this started failing? The `sqlstore` tests should have all been running against both MySQL and PostgreSQL before, right?

Can we pull this out into a separate PR to avoid this being buried?

Reply:
This is detected because the tests fails when you use postgres for app/api, that means something in the store layer is not 100% tested. Clearly postgres wasn't working in the 100% of the cases because the tests weren't passing for all the integration suite. Now they are showing up, but were there before.

Reply:
of course, I'm ok moving this to another PR.

Reply:
There are other plugin related bugs there, probably something to take a look.